scan/unstaged.go

```diff
@@ -80,7 +80,9 @@ func (us *UnstagedScanner) Scan() (Report, error) {
 				}
 			}
 
-			if _, err := io.Copy(workTreeBuf, workTreeFile); err != nil {
+			if fc, err := os.Readlink(fn); err == nil {
+				workTreeBuf = bytes.NewBufferString(fc)
```

Comment:
I think there is a bug here. We should read from target path to buffer instead of creating a buffer from path value.

Reply:
Sent PR with a fix: #634